lib/source.js

```diff
@@ -146,17 +165,19 @@ function originalEndPositionFor (sourceMap, line, column) {
     source: beforeEndMapping.source,
     line: beforeEndMapping.line,
     column: beforeEndMapping.column + 1,
-    bias: LEAST_UPPER_BOUND
+    bias: GREATEST_LOWER_BOUND
```

Comment:
Need to think about and double check this, kept getting confused..

but afact, LEAST_UPPER_BOUND is the left side of a position, GREATEST_LOWER_BOUND is the right